src/rpdk/guard_rail/rule_library/tags/schema-linter-core-tagging-rules.guard

```diff
@@ -144,6 +144,26 @@ rule ensure_property_tags_exists_v2 when tagging exists {
                 }
                 >>
             }
+            when createOnlyProperties exists {
+                tagging.tagProperty !IN createOnlyProperties
+                <<
+                {
+                    "result": "WARNING",
+                    "check_id": "TAG017",
+                    "message": "`tagProperty` MUST NOT be a part of `createOnlyProperties`"
+                }
+                >>
+
+                tagging.tagProperty !IN createOnlyProperties
+                tagging.tagUpdatable == false
```

Comment:
I am not very familiar with linter check rules, I thought it should be 
```
when tagging.tagProperty IN createOnlyProperties {
         tagging.tagUpdatable == false
```

Reply:
Updated